### ISSUE TITLE
Save State/ Pokemon Black Ferris Wheel Issue

### ISSUE THREAD

**Issue by tiltt350** [melonDS-emu/melonDS#2016]
Hey all, im very new here, and am having issues with my save file in pokemon black, i dont really know how to navigate issues with emulators at all and purely just want to be able to fix my game 

I am using my iphone 12 with delta installed and melonDS as the “core” with bios7.bin bios9.bin and firmware.bin, as mentioned in the title, my game wont save, ive been using save states to progress since the save stopped working, and have sinced logged many hours but am stuck at the ferris wheel where i know if u turn off the c-gear in the emulator beforehand it should all work fine, however because of my use of save states i cant turn it off where im at right now because it will prompt a saving of my game which will freeze my game again.

specific help and directions on this issue would be most greatly appreciated as im a hardcore beginner when it comes to anything emulator related haha

**Comment by nadiaholmquist** [comment]
Post your save state here.

**Comment by tiltt350** [comment]
![IMG_7533](https://github.com/melonDS-emu/melonDS/assets/167706080/42f320b7-5e93-4a23-97d2-9666f120c715)

![IMG_7534](https://github.com/melonDS-emu/melonDS/assets/167706080/f348fc72-4004-459f-b737-0ebfdaa33a0b)

![IMG_7532](https://github.com/melonDS-emu/melonDS/assets/167706080/be5aa394-35a0-400c-a5a8-fe01ec208dac)


the photos show what it says when i go to save, and what happens if i click yes, the other photo shows my array of save states

**Comment by nadiaholmquist** [comment]
You need to post the actual save state file. Those are a bit annoying to get at in Delta but you can probably find it.

**Comment by tiltt350** [comment]
i apologise for my ignorance, but im not quite sure what you meant 😭, I hope that this is sufficient, thank you for your patience 🙏 


[83E39C86-C760-4F82-91A6-655495974553.zip](https://github.com/melonDS-emu/melonDS/files/15098515/83E39C86-C760-4F82-91A6-655495974553.zip)

![83E39C86-C760-4F82-91A6-655495974553](https://github.com/melonDS-emu/melonDS/assets/167706080/80d96881-4fae-485b-8a9c-1ceb9c8b2266)

**Comment by nadiaholmquist** [comment]
No, I need the save state file. The one without the `.png`.

**Comment by jamesrath** [comment]
I'm having the same issue; my save state is frozen and the last in game save was 5 hours ago so I'm trying to figure out how I can recover my save state. It loads but the top screen is black and none of the buttons do anything. I was in the movie studio about to play a movie I just acted in. 

Here's a couple of the save states I'm trying to fix. I was hoping I could load them on my PC using melonDS and assuming it was able to work there save it and then send to my iPhone. 

[Saves.zip](https://github.com/melonDS-emu/melonDS/files/15133738/Saves.zip)

**Comment by nadiaholmquist** [comment]
[fixed.zip](https://github.com/melonDS-emu/melonDS/files/15137138/fixed.zip)

3 of the 4 states in your zip could be patched and I don't know which one is the most recent, so here are the 3 of them, you'll have to figure out which one is the right one yourself.

**Comment by Dom1248** [comment]
> [fixed.zip](https://github.com/melonDS-emu/melonDS/files/15137138/fixed.zip)
> 
> 
> 
> 3 of the 4 states in your zip could be patched and I don't know which one is the most recent, so here are the 3 of them, you'll have to figure out which one is the right one yourself.
> 
> 
> 
> Edit: As usual, turn off the C-Gear and keep it off

I am having the same exact problem, can you please fix mine😭😭🙏🏾 how do I send you my save state file?

**Comment by Dom1248** [comment]
And how will I load my fixed files into my game after they are fixed?

**Comment by RalphyUK** [comment]
I'd be extremely grateful if someone could help me with the same issue as above. Been stuck on the Ferris wheel cutscene for a while now and cannot find a fix. Very much appreciated in advance!

**Comment by nadiaholmquist** [comment]
You'll have to post your latest save state file here.

**Comment by RalphyUK** [comment]
![image](https://github.com/melonDS-emu/melonDS/assets/168590926/f76f133a-3027-4f01-af77-2b36d49dcbfa)

I'm struggling to download the save file. Any advice? 

Also thanks for replying so quickly !

**Comment by nadiaholmquist** [comment]
You need to get it from the Files app, under On My iPhone. Where exactly it is in the Delta folder can vary I think, their FAQ should tell you where to find it.

**Comment by RalphyUK** [comment]
![image](https://github.com/melonDS-emu/melonDS/assets/168590926/adaeb1c9-55e7-4317-b9be-424f6c484f47)

![image](https://github.com/melonDS-emu/melonDS/assets/168590926/819895f3-eebb-4732-ac6e-debd700e1ad0)

**Comment by RalphyUK** [comment]
This is where I'm at. I'm not doing very well :/

**Comment by nadiaholmquist** [comment]
You need to upload the one that is not a PNG. You may need to compress it for it to let you.

**Comment by RalphyUK** [comment]
[DC7E75FF-50BE-4A59-8257-CC77D7900AA1 11.zip](https://github.com/melonDS-emu/melonDS/files/15170870/DC7E75FF-50BE-4A59-8257-CC77D7900AA1.11.zip)

think this is wrong? thank you for your patience

**Comment by nadiaholmquist** [comment]
Nope, you compressed the PNG, it needs to be the file next to it with the same name.

**Comment by RalphyUK** [comment]
[saves pokemon black.zip](https://github.com/melonDS-emu/melonDS/files/15170992/saves.pokemon.black.zip)

i'm trying :)

**Comment by nadiaholmquist** [comment]
[Arkiv.zip](https://github.com/melonDS-emu/melonDS/files/15171030/Arkiv.zip)
You sent two states but one of them should be it.

**Comment by RalphyUK** [comment]
you're amazing, thank you. however, i have no idea how to get that save game and use it in the delta app on ios :(

just tried for 10 minutes and failing sadly.

i want to scream

**Comment by nadiaholmquist** [comment]
Replace the files where you found them with the ones from my zip, they need to have the same exact name. Note that you ended up with a (1) in one of them.

**Comment by RalphyUK** [comment]
this worked. thank you so much, i REALLY appreciate it. Hero!

**Comment by nadiaholmquist** [comment]
Great! Remember, don't turn on the C-Gear in your playthrough ever again or you will get the same crash.

**Comment by RalphyUK** [comment]
You're the best - thank you!

**Comment by DKismyhubby** [comment]
Omg @nadiaholmquist would you be willing to help another struggling soul? My husband is at his wits end because of the black screen

**Comment by DKismyhubby** [comment]
This is his file

[Uploading Dk_pokemon.zip…]()

**Comment by nadiaholmquist** [comment]
Hi, go to the issues section on this repository, and at the top there should be a PSA about the Pokémon issue and a link to a tool to fix it for you.

**Comment by aljwd4** [comment]
How can I save my linked save state to files? When I hold down on the load/save state, it only says “rename”, “lock”, “delete”, or “set as preview state”

**Comment by aljwd4** [comment]
When i go into my files to find my latest save state file, Im using delta and having trouble finding it. Can someone help me? It’s saying my zip file is too large when I try to post it here

**Comment by aljwd4** [comment]
I don’t even know how to link my latest save file

**Comment by akim12312** [comment]
@nadiaholmquist hi! would love some help as I am dealing with the same issue if you have time :,)

**Comment by jonahgoll** [comment]
@nadiaholmquist hello! I tried the JavaScript from the other thread but it did not work. I’ve attached my save file if you have the chance to look at it. I know I’ll need to turn off the c-gear as soon as I run the fixed file. Thanks!!!
[E2F5EC3C-4883-4DB5-AC9E-D4894A603A4C.zip](https://github.com/user-attachments/files/15928892/E2F5EC3C-4883-4DB5-AC9E-D4894A603A4C.zip)

**Comment by boostedbuddha** [comment]
> [Arkiv.zip](https://github.com/melonDS-emu/melonDS/files/15171030/Arkiv.zip) You sent two states but one of them should be it.

Would you be able to help me do mine as well? 
![6349E505-F2AB-4287-8418-44F3550D02B7 2](https://github.com/melonDS-emu/melonDS/assets/175306741/fce05fb8-f259-4ff9-a04a-cf117bc251cc)

**Comment by boostedbuddha** [comment]
I realize that I sent a png, how would I access the real save state file to send it? My iphone won’t let me send it at all here
